pytest_fixtures/xdist.py

```diff
@@ -0,0 +1,13 @@
+"""Fixtures specific to or relating to pytest's xdist plugin"""
+import pytest
+
+from robottelo.config import settings
+
+
+@pytest.fixture(scope="session", autouse=True)
+def align_xdist_satellites(worker_id):
+    """Set a different Satellite per worker when available in robottelo's config"""
+    settings.configure()
```

Comment:
Please check if its already configured, if these is an inbuilt check fr it then i m all good fr it.

Reply:
settings.configure has an internal check for this